src/tests/readtime.test.js

```diff
@@ -23,6 +23,11 @@ describe('READ TIME FUNCTIONS', () => {
     expect(textTime).to.eql(459);
   });
 
+  it('should return time it takes to read text in seconds', () => {
+    const textTime = calcTextTime('words');
+    expect(textTime).to.eql(0);
```

Comment:
From the test description, this should return time in seconds.

Reply:
The description has been changed to reflect the purpose of the test